商品削除機能の実装

tomo1107 opened the pull request:
# What
## 実装内容
- 商品削除機能の実装

## 動作確認結果(Gyazo画像)
- ログイン状態の出品者のみ、詳細ページの削除ボタンを押すと、出品した商品を削除できる動画
https://gyazo.com/b9f290a477f9fc69914ab0296bc93651

# Why
- 商品削除機能が正常に実装出来ているかをご確認お願いいたします。

tomo1107 committed:
商品削除機能の実装

Hide-Morishita commented on config/routes.rb:
```diff
@@ -1,5 +1,5 @@
   devise_for :users
   root to: "items#index"
-  resources :items, only: [:new, :create, :show, :edit, :update]
+  resources :items, only: [:new, :create, :show, :edit, :update, :destroy]
```
```ruby
resources :items, only: [:new, :create, :show, :edit, :update, :destroy]
```
7つのアクション全て揃いましたので、,only:~~の部分は削除しましょう！

Hide-Morishita reviewed (commented):
レビュー依頼、キャプチャ動画ありがとうございます。
コメントいたしましたので、ご確認をお願いいたします。

tomo1107 committed:
商品削除機能の実装(是正1回目)

以下のご指摘に対しての是正対応
https://github.com/tomo1107/furima-35073/pull/7#pullrequestreview-636494827

tomo1107 commented:
> レビュー依頼、キャプチャ動画ありがとうございます。
> コメントいたしましたので、ご確認をお願いいたします。

ご確認ありがとうございました！
ご指摘頂いた箇所を是正しましたので、再レビューの程、よろしくお願いいたします！

ghost commented:
レビュー依頼ありがとうございます！
LGTMとさせていただきます！実装お疲れ様でした！
このブランチをマージし、masterブランチでプルを行いましょう！
機能が反映されていることを確認し、引き続き実装頑張っていきましょう！
